### PR TITLE
T35143 Enable pylint on  api.main

### PR DESCRIPTION
**Pull request by JenySadadia**
api/main.py: fix pylint errors
api/main.py: update copyright header
github: enable pylint on api.main

### CODE REVIEW

**Comment by JenySadadia** [comment]
@gctucker Currently, I have disabled "global-statement" error. `pubsub` is causing it.
One way to resolve it is to declare a global dict for it.
`pubsub = {}`

File: api/main.py [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
 from .pubsub import PubSub, Subscription
-from typing import List
-from bson import ObjectId, errors
 
 app = FastAPI()
 db = Database()
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
 auth = Authentication(db)
-pubsub = None
+PUBSUB = None
```
Comment (mgalka) [inline api/main.py:24]: Why do we have this name in capitals?
PEP8 suggests that capitals should be used when we're declaring a  constant.

Comment (JenySadadia) [reply api/main.py:24]: > Why do we have this name in capitals? PEP8 suggests that capitals should be used when we're declaring a constant.

Pylint was complaining about it with "invalid-name" error.

Comment (mgalka) [reply api/main.py:24]: Can you link the pylint logs?

Comment (JenySadadia) [reply api/main.py:24]: Here are the logs:
```
api/main.py:19:0: C0103: Constant name "pubsub" doesn't conform to UPPER_CASE naming style (invalid-name)
api/main.py:24:4: C0103: Constant name "pubsub" doesn't conform to UPPER_CASE naming style (invalid-name)
```

Comment (mgalka) [reply api/main.py:24]: It's definitely a variable, hence it shouldn't be all capitals. I think it's reasonable to add `pubsub` to `good-names` in pylint configuration or simply turn off the warning for this particular line with a `# pylint: disable=C0103` comment

Comment (JenySadadia) [reply api/main.py]: Done.

**Comment by JenySadadia** [comment]
Rebased the PR.

**Comment by gctucker** [comment]
Please rebase.

@mgalka Was this PR ready to be merged otherwise?

**Comment by mgalka** [comment]
> Please rebase.
> 
> @mgalka Was this PR ready to be merged otherwise?

yes

**Comment by JenySadadia** [comment]
> Please rebase.
> 
> @mgalka Was this PR ready to be merged otherwise?

Done.

**Pull request merged**